static/app/routes.tsx

```diff
@@ -1253,12 +1253,11 @@ function buildRoutes() {
 
   const issueListRoutes = (
     <Route
-      path="/organizations/:orgId/issues/"
+      path="/organizations/:orgId/issues/(searches/)(:searchId/)"
```

Comment:
Should it be 
```suggestion
      path="/organizations/:orgId/issues/(searches/:searchId/)"
```

is there a `/organizations/:orgId/issues/searches/`?

Reply:
Good catch, I think you're right